src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -117,7 +117,8 @@ export function loadStateFromAuthoring(state, authoring, progress={}) {
         numTrials = authoredChallenge.numTrials || (trials ? trials.length : 1),
         trialOrder = createTrialOrder(trial, trials, state.trialOrder, authoredChallenge.randomizeTrials),
         drakes = processAuthoredDrakes(authoredChallenge, trialOrder[trial], template, trial),
-        gametes = processAuthoredGametes(authoredChallenge, drakes, state);
+        gametes = processAuthoredGametes(authoredChallenge, drakes, state),
+        zoomUrl = authoredChallenge.zoomUrl || null;
```

Comment:
Looks like other undefined pieces of the authoring doc stay `undefined` rather than `null` - what's the reasoning for setting the `zoomUrl` to `null` by default?